### PR TITLE
Add: track event to control link.

### PR DESCRIPTION
**Pull request by teesloane**
https://github.com/circleci/circleci-docs/pull/6447 did not add track control events. This PR adds that.

### CODE REVIEW

**Comment by tshen815** [comment]
Is this specifically to track an event when users are segmented into the control group? Would we need to gate the new trackAction you made in a conditional? 

something like:
`if (variation === 'control') {// new trackAction for control events here} `

**Review by tshen815 — COMMENTED** [review]
Is this specifically to track an event when users are segmented into the control group? Would we need to gate the new trackAction you made in a conditional? 

something like:
`if (variation === 'control') {// new trackAction for control events here} `

**Comment by teesloane** [comment]
> Is this specifically to track an event when users are segmented into the control group? Would we need to gate the new trackAction you made in a conditional?
> 
> something like: `if (variation === 'control') {// new trackAction for control events here} `

Oh, good catch, I will move that into a control block.

File: src/js/experiments/signupLinksCta.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ function handleClickedLink() {
   });
 }
 
+
```
Comment (rsylvian) [inline src/js/experiments/signupLinksCta.js:34]:
```suggestion
```

**Review by chantelle11 — APPROVED** [review]
Nice! 👍😁😊

**Review by tshen815 — APPROVED** [review]
looks good!

**Pull request merged**